bpytop.py

```diff
@@ -3069,10 +3074,22 @@ def _collect(cls):
 					errlog.error(f'Caused by outdated psutil version.')
 				errlog.exception(f'{e}')
 			io_counters = None
+		try:
```

Comment:
Should probably add some conditions to this block since it doesn't need to run if `CONFIG.use_fstab == False` and also if `SYSTEM == "MacOS"` since OsX don't use `/etc/fstab`.

For performance you could possibly store `fstab_filter` as a class variable also and only run the code block if the list is empty. `/etc/fstab` should be unlikely to change right?

Reply:
Hi, thanks for the review, I will be updating for optimization.
My first idea was to hide the option if `/etc/fstab` does not exist, sticking it to `False`, however if I find no clean way to do this, I'll keep on your idea.
Agreed about the refresh based on `/etc/fstab` changes, it was rather a test case for me.